internal/extractors/extract_data_test.go

```diff
@@ -68,13 +68,13 @@ func Example() {
 	// CCD3  got stop packet without a start packet
 	// Unknown RID: 303  unhandled RID Unknown RID: 303
 	//  STAT <nil>
-	//   the TMHeader isn't recognized as either housekeeping or transparent data
-	//   the TMHeader isn't recognized as either housekeeping or transparent data
+	//   the TMHeader isn't recognized as either housekeeping, transparent or verification data
```

Comment:
FYI, this was really not a proper output here and my PR should fix it. It's just that errors that originated at a earlier stage were ignored and overwritten.